deployment/tfhub/object-detection-url.ipynb

```diff
@@ -0,0 +1,625 @@
+      "source": [
+        "endpoint = client.get_or_create_endpoint('object-detection-url')"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "colab": {
+          "base_uri": "https://localhost:8080/"
+        },
+        "id": "F4-9M2_xKF7R",
+        "outputId": "acf0965c-da8b-442b-f170-a9af8c2255bf"
+      },
+      "outputs": [],
+      "source": [
+        "%%time\n",
+        "endpoint.update(\n",
+        "    model, \n",
+        "    strategy = DirectUpdateStrategy(),\n",
+        "    autoscaling = autoscaling,\n",
+        "    resources = resources,\n",
+        "    wait = True\n",
+        ")"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "0Zgc18ocXbCE"
+      },
+      "outputs": [],
+      "source": [
+        "deployed_model = endpoint.get_deployed_model()"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "NbxYiJJsX5oL"
+      },
+      "source": [
+        "## 8. Predictions"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "p7x7dfWoCNxK"
+      },
+      "outputs": [],
+      "source": [
+        "url = 'http://www.cs.ucf.edu/~aroshan/index_files/Dataset_PitOrlManh/images/'"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "4nS6rzzyK58c"
+      },
+      "outputs": [],
+      "source": [
+        "req = requests.get(url)"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "gH_5iPC84jmu"
+      },
+      "outputs": [],
+      "source": [
+        "soup = BeautifulSoup(req.text, 'lxml')"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "Hl_MzSFXX2oo"
+      },
+      "outputs": [],
+      "source": [
+        "urls = []\n",
+        "\n",
+        "for link in soup.find_all('a'):\n",
+        "    href = link.get('href')\n",
+        "  \n",
+        "    if href.endswith('.jpg'):\n",
+        "        image_url = f\"{url}/{href}\"\n",
+        "        urls.append(image_url)"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "Ifi5nJeBX_1G"
+      },
+      "outputs": [],
+      "source": [
+        "urls = urls[:10]"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {},
+      "outputs": [],
+      "source": [
+        "urls = [[os.path.basename(url), url] for url in urls]"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "6puNawPEYBJl"
+      },
+      "outputs": [],
+      "source": [
+        "def process_image(url):\n",
+        "    return deployed_model.predict([url])"
```

Comment:
Since we are sending only one `url` at a time, we can do a `[0]` here to get `result` instead of `[result]`.

```suggestion
        "    return deployed_model.predict([url])[0]"
```

Reply:
Done.